best-practices-trove-project-creation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Creating projects for Trove"
+permalink: /best-practices-trove-project-creation/
+---
+# Creating a data collection project in Trove
+
+With Trove, you can create AI/ML data collection projects through trusted connections between you and the people who contribute to your projects, leading to higher quality data. This document explains how to create such a project in Trove.
```

Comment:
This is our opportunity to tell people more about what projects they can launch. We only support photos right now and we should make sure people that read this understand that. I would be specific about computer vision and photos.

Reply:
This opening sentence was taken from paragraph for AI developers on our marketing page at https://www.microsoft.com/en-us/ai/trove-images-for-machine-learning but I agree, we should be specific about CV and photos for now.